components/src/ProductListPagination/productlistpagination.main.tsx

```diff
@@ -52,120 +42,39 @@ class ProductListPagination extends React.Component<ProductListPaginationProps,
       search: '',
       category: '',
     },
-  }
+  };
 
   constructor(props) {
     super(props);
     const { paginationDataProps } = this.props;
     ({ intl } = getConfig());
-    this.state = {
-      paginationData: paginationDataProps,
-      paginationPreviousLink: '',
-      paginationNextLink: '',
-      searchUrl: true,
-    };
-  }
-
-  componentDidMount() {
-    paginationPreviousLinkVar = '';
-    paginationNextLinkVar = '';
-    searchUrlVar = true;
-    this.setState({
-      paginationPreviousLink: paginationPreviousLinkVar,
-      paginationNextLink: paginationNextLinkVar,
-      searchUrl: searchUrlVar,
-    });
+    this.state = { paginationData: paginationDataProps };
```

Comment:
This is outside of scope of this PR but we shouldn't be storing parent props in a state. That is not a valid React pattern. You don't have to fix it for this PR but we should change this at some point.

Reply:
Thank for the comment, I'll update it here so we don't need to come back to it later.